### PR TITLE
chore: add new quote

### PR DESCRIPTION
**Pull request by whydnxx**

### CODE REVIEW

**Comment by whydnxx** [comment]
Hi @nyancodeid 

I'm currently participating in the hacktoberfest event, could you add the label "hacktoberfest-accepted" for me? thanks

File: quotes/whydnxx.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+username: whydn.dev
```
Comment (nyancodeid) [inline quotes/whydnxx.yaml:1]: Can we have same github username, just like filename and your github username. And also the author name should your real name or github username. Thank you @whydnxx

**Review by nyancodeid — CHANGES_REQUESTED** [review]
Need some changes

**Review by whydnxx — COMMENTED** [review]
Thanks for remind me, I've change the username

**Comment by nyancodeid** [comment]
Awesome, thank you @whydnxx

**Pull request merged**